src/testsuite/arith128_test_bcd.c

```diff
@@ -722,6 +722,175 @@ db_vec_cbcdsubcsq (vBCD_t *cout, vBCD_t a, vBCD_t b)
   return (t);
 }
 
+/* Convert radix 100 binary bytes to radix 10 BCD Nibbles */
+vui8_t
+db_vec_rdxcf100b (vui8_t vra)
+{
+  vui8_t result;
+  vui8_t x6, c6, high_digit;
+  /* Compute the high digit correction factor. For binary 100s to BCD
+   * this is the radix 100 value divided by 10 time by the radix
+   * difference in binary.  For this stage we use 0x10 - 10 = 6.  */
+  print_vint8d ("16xrdx100  ", vra);
+  print_vint8x ("           ", vra);
+  high_digit = vra / 10;
+  print_vint8d (" high digit", high_digit);
+  c6 = vec_splat_u8 ((unsigned char) 0x06);
+  x6 = vec_mulubm (high_digit, c6);
+  print_vint8d (" correction", x6);
+  /* Add the high digit correction bytes to the original
+   * radix 100 bytes in binary. */
+  result = vec_add (vra, x6);
+  return result;
+}
+
+/* Convert radix 10**4 binary hwords to radix 100 bytes */
+vui8_t
+db_vec_rdxcf10kh (vui16_t vra)
+{
+  vui8_t result;
+  vui16_t x156, c156, high_digit;
+  /* Compute the high digit correction factor. For binary 10**4 to 100s
+   * this is the radix 10000 value divided by 100 times by the radix
+   * difference in binary.  For this stage we use 0x100 - 100 = 156.  */
+  print_vint16d ("8xrdx10k   ", vra);
+  print_vint16x ("           ", vra);
+  high_digit = vra / 100;
+  print_vint16d (" high digit", high_digit);
+  c156 = vec_splats ((unsigned short) 156);
+#if 0
+  x156 = vec_muluhm (high_digit, c156);
+#else
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+  x156 = vec_vmuleub ((vui8_t) high_digit, (vui8_t) c156);
+#else
+  x156 = vec_vmuloub ((vui8_t) high_digit, (vui8_t) c156);
+#endif
+#endif
+  print_vint16d (" correction", x156);
+  /* Add the high digit correction hword to the original
+   * radix 10000 hword in binary. */
+  result = (vui8_t) vec_add (vra, x156);
+  return result;
+}
+
+/* Convert radix 10**8 binary words to radix 10**4 hwords */
+vui16_t
+db_vec_rdxcf100mw (vui32_t vra)
+{
+  vui16_t result;
+  vui32_t x, c, high_digit;
+  /* Compute the high digit correction factor. For binary 10**8 to 10**4
+   * this is the radix 100000000 value divided by 10000 times by the radix
+   * difference in binary.  For this stage we use 0x10000 - 10000 = 55536.  */
+  print_vint32d ("4xrdx100m  ", vra);
+  print_vint32x ("           ", vra);
+  high_digit = vra / 10000;
+  print_vint32d (" high digit", high_digit);
+  c = vec_splats ((unsigned int) 55536);
+#if 0
+  x = vec_muluwm (high_digit, c);
```

Comment:
do you want to keep this?
I'll stop flagging the code that is `#if 0` protected... there are many.

Reply:
No, deleted